src/components/Banner/HeroBanner/HeroBanner.jsx

```diff
@@ -0,0 +1,11 @@
+import "./heroBanner.css";
+
+function HeroBanner() {
+  return (
+    <section class="homepage_banner_container">
+      <div class="banner_content_container"></div>
```

Comment:
Change your class to className everywhere.

Reply:
Done